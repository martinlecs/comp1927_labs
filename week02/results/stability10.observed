1 yxz
1 xyz
2 lmn
2 nnl
3 bba
3 bab
6 ddd
6 aaa
7 bbb
9 ccc
