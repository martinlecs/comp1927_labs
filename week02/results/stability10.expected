1 xyz
1 yxz
2 lmn
2 nnl
3 bab
3 bba
6 aaa
6 ddd
7 bbb
9 ccc
